frontend/src/components/admin/UsersTable.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { Icon } from "@chakra-ui/icon";
+import { MdDelete } from "react-icons/md";
+import {
+  Badge,
+  Link,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  IconButton,
+} from "@chakra-ui/react";
+import { User } from "../../APIClients/queries/UserQueries";
+import convertLanguageTitleCase from "../../utils/LanguageUtils";
+
+export type UsersTableProps = {
+  isTranslators?: boolean;
+  users: User[];
+};
+
+const UsersTable = ({ isTranslators = false, users }: UsersTableProps) => {
+  const approvedLanguanges = users.map((userObj: User) => {
+    const approvedLanguange = isTranslators
+      ? userObj?.approvedLanguagesTranslation
+      : userObj?.approvedLanguagesReview;
+    return approvedLanguange
+      ? JSON.parse(approvedLanguange!!.trim().replace(/'/g, '"'))
+      : "";
+  });
+
+  const generateBadges = (approvedLanguages: any) =>
+    Object.keys(approvedLanguages).map((apprLang: string) => (
+      // TODO: determine badge color by label
+      <Badge background="orange.50">{`${convertLanguageTitleCase(
```

Comment:
the badges on the table seem a lot bolder than the badges in the figma mockup (i think especially since each language is in all caps)--is this on purpose?
 
![image](https://user-images.githubusercontent.com/32009013/137339763-fc5adb1a-43e9-4cb5-b352-de8c73f15ff3.png)